documents/invoice/model.go

```diff
@@ -21,7 +21,7 @@ import (
 
 const prefix string = "invoice"
 
-var compactPrefix = []byte{1, 0, 0, 0}
+func compactPrefix() []byte { return []byte{1, 0, 0, 0} }
```

Comment:
I think it would be nice to move this to centrifuge-protobufs: https://github.com/centrifuge/centrifuge-protobufs/blob/master/documenttypes/prototypes.go